test/testcafe/spec/IdentifyOktaVerify_spec.js

```diff
@@ -52,8 +54,9 @@ test('should the correct title', async t => {
 test('should the correct content', async t => {
   const identityPage = await setup(t);
   await t.expect(identityPage.getFormTitle()).eql('Sign In');
-  await t.expect(identityPage.getOktaVerifyButtonText()).eql('Sign in with Okta FastPass');
-  await t.expect(identityPage.getSeparationLineText()).eql('or');
+  await t.expect(identityPage.getOktaVerifyButtonText()).contains('Sign in with Okta FastPass');
+  const separationText = await identityPage.getSeparationLineText();
+  await t.expect(separationText.toString().toLowerCase()).eql('or');
```

Comment:
We changed the capitalization causing this to fail? I think this element deserves some attention from design